src/middlewares/authentication.js

```diff
@@ -0,0 +1,77 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+import Models from '../models';
+
+const { SECRET_KEY } = process.env;
+const { BlacklistedTokens } = Models;
+
+config();
+
+/**
+ *
+ *
```

Comment:
what are these empty spaces for in the jsdoc comment?

Reply:
Fixed that. Thanks